<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageCalendar">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pageDial">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QDial" name="dial"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>92</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="calendarPageBtn">
       <property name="text">
        <string>日历界面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dialPageBtn">
       <property name="text">
        <string>刻度盘界面</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
